### DISCUSSION TITLE
宣传一下桌宠模拟器的自制Mac版

### DISCUSSION THREAD

**Discussion by justaLoli** [LorisYounger/VPet#195] [Show and tell]
由于全无开发经验，项目很粗糙，我需要友友帮忙测试并且反馈一些bug🥺

用Mac的朋友可以下载玩玩嘛？

https://github.com/justaLoli/VPet-Mac

目前功能包括：

✅ 开始、关闭、正常效果的动画播放；

✅ 拖动效果；

✅ 「互动」菜单里的互动，即睡觉、学习、工作等（不过目前没有显示计时器，也没有经验、金钱加成）

✅ 自动事件（发呆、待机、睡觉等）；

✅ 桌宠自动移动。

已实现功能和原版基本类似，但随机事件的触发逻辑可能不同。

由于开发能力限制，其他功能开发会放缓。

**Comment by BeiMO0192** [comment]
我是Mac用户，对于这个软件，你做的很不错，我很喜欢，加油
